courier/courier.go

```diff
@@ -49,11 +49,12 @@ type (
 	}
 
 	courier struct {
-		smsClient   *smsClient
-		smtpClient  *smtpClient
-		deps        Dependencies
-		failOnError bool
-		backoff     backoff.BackOff
+		smsClient    *smsClient
+		smtpClient   *smtpClient
+		mailerClient *mailerClient
```

Comment:
Could we name this `httpClient` instead? The `smtpClient` also refers to the protocol it's using, and `mailer` is arguably a bit ambiguous.

Reply:
+1 ... I thought the same